test/data/users.js

```diff
@@ -24,7 +24,11 @@ export const extendedUsers = dataStore({
     role = 'admin',
     verbs = verbsByRole(role),
     createdAt = undefined,
-    deletedAt = undefined
+    deletedAt = undefined,
+    preferences = {
+      site: {},
+      projects: {},
+    },
```

Comment:
Rather than expecting the full `preferences` object here, maybe it would be useful to allow tests to specify a partial object, then merge that partial object with the full object below. So here, it would be:

```js
preferences = {},
```

And below, it would be:

```js
// mergeDeepLeft() is from ramda
preferences: mergeDeepLeft(preferences, {
  site: {},
  projects: {}
}),
```

That would allow a test to easily specify a site preference without also specifying `projects: {}`, e.g., something like:

```js
// test/components/project/list.spec.js
it('uses the projectSortMode preference', () => {
  // Specifying site without projects.
  mockLogin({ preferences: { site: { projectSortMode: 'alphabetical' } } });
  createProjects([
    { name: 'A' },
    { name: 'B', lastSubmission: ago({ days: 5 }).toISO() }
  ]);
  const component = mountComponent();

  const blocks = mountComponent().findAllComponents(ProjectHomeBlock);
  // ['B', 'A'] is the default, as B has the latest data.
  blocks.map(block => block.props().project.name).should.eql(['A', 'B']);

  const select = component.getComponent(ProjectSort);
  select.props().modelValue.should.equal('alphabetical');
});
```

Just an idea! Could also come later once tests are added.